fr/_comments.md

```diff
@@ -0,0 +1,233 @@
+
+
+
+### Lecture complémentaire
+
+* [CSS Guidelines' Commenting section](http://cssguidelin.es/#commenting)
+
+{% include donate.html %}
+
+
+
+
+
+
+## Documentation
+
+Chaque variable, fonction, mixin et placeholder destiné à être réutilisé dans le code devrait être documenté en tant que partie de l'API globale à l'aide de [SassDoc](http://sassdoc.com).
+
+SassDoc propose deux syntaxes différentes pour les commentaires, le style C ou en ligne. Par exemple, les deux extraits de code suivants sont des commentaires valables pour SassDoc&nbsp;: 
+
+<div class="code-block">
+  <div class="code-block__wrapper" data-syntax="scss">
+{% highlight scss %}
+/**
+ * Ligne de base du rythme vertical utilisé sur l'ensemble du code.
+ * @type Length
+ */
+$vertical-rhythm-baseline: 1.5rem;
+{% endhighlight %}
+  </div>
+  <div class="code-block__wrapper" data-syntax="sass">
+{% highlight sass %}
+/**
+ * Ligne de base du rythme vertical utilisé sur l'ensemble du code.
+ * @type Length
+ */
+$vertical-rhythm-baseline: 1.5rem
+{% endhighlight %}
+  </div>
+</div>
+
+<div class="code-block">
+  <div class="code-block__wrapper" data-syntax="scss">
+{% highlight scss %}
+/// Ligne de base du rythme vertical utilisé sur l'ensemble du code.
+/// @type Length
+$vertical-rhythm-baseline: 1.5rem;
+{% endhighlight %}
+  </div>
+  <div class="code-block__wrapper" data-syntax="sass">
+{% highlight sass %}
+/// Ligne de base du rythme vertical utilisé sur l'ensemble du code.
+/// @type Length
+$vertical-rhythm-baseline: 1.5rem
+{% endhighlight %}
+  </div>
+</div>
+
+<div class="note">
+  <p>Trois barres obliques (<code>/</code>) sont requises.</p>
+</div>
+
+Il n'y a pas d'avantage particulier à utiliser l'une ou l'autre méthode, choisissez celle avec laquelle vous vous sentez le plus à l'aise.
+
+SassDoc a deux rôles principaux&nbsp;:
+
+* imposer une standardisation des commentaires utilisant un système basé sur les annotations, pour tout ce qui fait partie d'une API publique ou privée&nbsp;; 
+* permettre de générer une version HTML de la documentation API en utilisant n'importe quel endpoint SassDoc (CLI tool, Grunt, Gulp, Broccoli, Node...).
+
+<figure role="group">
+<img alt="Documentation générée par SassDoc" src="http://sassdoc.com/develop/assets/images/preview-image.png" />
+<figcaption>Documentation générée par SassDoc</figcaption>
+</figure>
+
+Voici un exemple de mixin extensivement documenté avec SassDoc&nbsp;:
+
+<div class="code-block">
+  <div class="code-block__wrapper" data-syntax="scss">
+{% highlight scss %}
+/// Mixin servant à définir `width` et `height` simultanément.
+///
+/// @author Hugo Giraudel
+///
+/// @access public
+///
+/// @param {Length} $width - Largeur `width` de l'élément
+/// @param {Length} $height ($width) - Hauteur `height` de l'élément
+///
+/// @example scss - Utilisation
+/// .foo {
+///   @include size(10em);
+/// }
+///
+/// .bar {
+///   @include size(100%, 10em);
+/// }
+///
+/// @example css - CSS après compilation
```

Comment:
Please, add a 2-spaces indent to content of `@example` (+1 space to prevent the content from being stuck to `///`) such as:

```scss
/// @example css - CSS après compilation
///   .foo {
///     width: 10em;
///     height: 10em;
///   }
```